codepipeline-tf/providers.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 1.0"

```

Comment:
can we use latest version with proper constraints

Reply:
upgraded to 1.7.